vetiver/server.py

```diff
@@ -129,20 +142,23 @@ def vetiver_post(self, endpoint_fx: Callable, endpoint_name: str = None, **kw):
         >>> import vetiver as vt
         >>> X, y = vt.get_mock_data()
         >>> model = vt.get_mock_model().fit(X, y)
-        >>> v = vt.VetiverModel(model = model, model_name = "model", ptype_data = X)
-        >>> v_api = vt.VetiverAPI(model = v, check_ptype = True)
+        >>> v = vt.VetiverModel(model = model, model_name = "model", prototype_data = X)
+        >>> v_api = vt.VetiverAPI(model = v, check_prototype = True)
         >>> def sum_values(x):
         ...     return x.sum()
         >>> v_api.vetiver_post(sum_values, "sums")
         """
         if not endpoint_name:
             endpoint_name = endpoint_fx.__name__
 
-        if self.check_ptype is True:
+        if hasattr(self.model, "ptype"):
```

Comment:
might need to be stricter? I don't think it's possible to have a prototype and a ptype that would be two different things, unless someone is doing something wrong, but maybe make sure ptype is not None/the same as prototype

Reply:
I don't see how that could happen either.

Reply:
Ah, that's a good point. I'll update it to reassign legacy ptype to prototype, and then remove the ptype attribute